docs/integrations/prometheus.md

```diff
@@ -32,7 +32,7 @@ clusters:
 # We created a namespace called "default" and had set it to retention "48h".
        - namespace: default
          retention: 48h
-         storageMetricsType: unaggregated
+         type: unaggregated
```

Comment:
is this going to be b/w compatible? main concern is if skipping specifying this field be the same as `unaggregated`?